src/main/java/dl/news/portal/domain/service/impl/RefreshTokenServiceImpl.java

```diff
@@ -0,0 +1,75 @@
+package dl.news.portal.domain.service.impl;
+
+import dl.news.portal.domain.entity.RefreshToken;
+import dl.news.portal.domain.entity.User;
+import dl.news.portal.domain.repository.RefreshTokenRepository;
+import dl.news.portal.domain.service.RefreshTokenService;
+import dl.news.portal.domain.service.UserService;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.stereotype.Service;
+
+import javax.persistence.EntityNotFoundException;
+import java.util.Date;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Service
+public class RefreshTokenServiceImpl implements RefreshTokenService {
+
+    private static final long REFRESH_TOKEN_VALIDITY_SECONDS = 5 * 60 * 60 * 24;
```

Comment:
What does this numbers means?

Reply:
Time in seconds while refresh token is valid